src/_posts/languages/java/2000-01-01-start.md

```diff
@@ -16,20 +16,21 @@ The following Java versions are available:
 
 | Java SE Version | scalingo-20       | scalingo-22       |
 | --------------: | ----------------: | ----------------: |
-| **`7`**         | up to `1.7.0_352` | up to `1.7.0_352` |
-| **`8 (LTS)`**   | up to `1.8.0_422` | up to `1.8.0_422` |
-| **`10`**        | up to `10.0.2`    | up to `10.0.2`    |
-| **`11 (LTS)`**  | up to `11.0.24`   | up to `11.0.24`   |
-| **`13`**        | up to `13.0.14`   | up to `13.0.14`   |
-| **`14`**        | up to `14.0.2`    | up to `14.0.2`    |
-| **`15`**        | up to `15.0.10`   | up to `15.0.10`   |
-| **`16`**        | up to `16.0.2`    | up to `16.0.2`    |
-| **`17 (LTS)`**  | up to `17.0.12`   | up to `17.0.12`   |
-| **`18`**        | up to `18.0.2.1`  | up to `18.0.2.1`  |
-| **`19`**        | up to `19.0.2`    | up to `19.0.2`    |
-| **`20`**        | up to `20.0.2`    | up to `20.0.2`    |
-| **`21 (LTS)`**  | up to `21.0.4`    | up to `21.0.4`    |
+| **`23`**        | up to `23.0.1`    | up to `23.0.1`    |
 | **`22`**        | up to `22.0.2`    | up to `22.0.2`    |
+| **`21 (LTS)`**  | up to `21.0.5`    | up to `21.0.5`    |
+| **`20`**        | up to `20.0.2`    | up to `20.0.2`    |
+| **`19`**        | up to `19.0.2`    | up to `19.0.2`    |
+| **`18`**        | up to `18.0.2.1`  | up to `18.0.2.1`  |
+| **`17 (LTS)`**  | up to `17.0.13`   | up to `17.0.13`   |
+| **`16`**        | up to `16.0.2`    | up to `16.0.2`    |
+| **`15`**        | up to `15.0.10`   | up to `15.0.10`   |
+| **`14`**        | up to `14.0.2`    | up to `14.0.2`    |
+| **`13`**        | up to `13.0.14`   | up to `13.0.14`   |
+| **`11 (LTS)`**  | up to `11.0.25`   | up to `11.0.24`   |
+| **`10`**        | up to `10.0.2`    | up to `10.0.2`    |
+| **`8 (LTS)`**   | up to `1.8.0_432` | up to `1.8.0_432` |
+| **`7`**         | up to `1.7.0_352` | up to `1.7.0_352` |
```

Comment:
quesiton: What do you think about removing the versions that are deprecated (https://www.oracle.com/java/technologies/java-se-support-roadmap.html)?

Reply:
I kept the versions for which the Premier Support and Extended Support are still valid. Would that work for you?